pkgs/misc/base16-shell-preview/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "base16_shell_preview";
+  version = "0.3.0";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0x2fbicrcqgf2dl7dqzm14dz08vjjziabaaw33wah3v9wv4rw7jq";
+  };
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Browse and preview Base16 Shell themes in your terminal.";
```

```suggestion
    description = "Browse and preview Base16 Shell themes in your terminal";
```

No periods in `description`.
> https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes